testnet-contracts/migrations/2_next.js

```diff
@@ -81,7 +90,7 @@ module.exports = function(deployer, network, accounts) {
     // 1. Deploy BridgeToken contract
     //    Gas used:        1,884,394 Gwei
     //    Total cost:    0.03768788 Ether
-    await deployer.deploy(BridgeToken, "TEST", {
+    await deployer.deploy(BridgeToken, "stake", {
```

Comment:
why is this called stake? isn't this just a dummy ERC-20 to be used as an Eth based asset that is moved to the cosmos chain?

Reply:
Ah, good catch. While attempting to end-to-end test around the issues with whitelisting native cosmos assets described in https://github.com/cosmos/peggy/issues/123 I updated the testing token's denom to stake. Fixed token denom in https://github.com/cosmos/peggy/pull/160/commits/b5b5d7d47d7531d2b7420dcb8da450a37bb917aa.

The whitelisting bug will be addressed within a few days as part of updating namespacing.